acmwebsite/templates/survey_results.xhtml

```diff
@@ -13,19 +13,19 @@
     <div class="row">
       <div class="col-md-12">
         <table class="table table-striped table-condensed table-bordered">
-          <!--
+          <!--!
           I'm doing some fancy shenanigans here to allow the user to sort by
           each column ascending and descending.
           -->
           <thead>
             <tr>
               <th>
```

Comment:
repeated code here... just put it as the first thing you iterate in the `py:for` below.